app/components/attachment_button/index.test.js

```diff
@@ -4,10 +4,10 @@
 import React from 'react';
 import {Alert, StatusBar} from 'react-native';
 import Permissions from 'react-native-permissions';
+import {shallowWithIntl} from 'test/intl-test-helper';
```

Comment:
@larkox  Let's create an alias for the `test` directory.  Just to prevent the unintentional sorting at line 7.